app/helpers/ct_table_for/application_helper.rb

```diff
@@ -81,6 +83,11 @@ def table_for_content model, collection, options: {}
       html.html_safe
     end
 
+    def data_href_for(record, options)
+      return unless options[:clickable]
+      "data-href=" << (options[:row_uri] && uri?(options[:row_uri]) ? options[:row_uri] : polymorphic_url(record))
```

Comment:
This line is not enough dynamic. Some comments about it:
* Using ```:row_uri``` with an static value means that every row will have the same href.
* ```polymorphic_url``` works if exist a direct url. For example ```user_path```. But when you're inside some namespace, like ```bo_user_path```, this line does not work. Or if the object needs a parent object.

```polymorphic_url``` method admits objects and symbols as first params to fullfil complex paths. So, I would change ```row_uri``` to a some sort of multipurpose param, that could handle different types of inputs. Static values are not a suitable solution.